starter_code/src/movies.js

```diff
@@ -1,20 +1,99 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
-
```

Comment:
So your code works if I, for example, explicitly enter a duration of a film. However, I'd like to see the a new array with all the movies and only the duration is changed from hours to minutes.

Reply:
function turnHoursToMinutes(arr){
    for(i of arr){
      i.duration = hourtominute(i.duration)
    }
    return(arr)
  }

its right below it :)